### ISSUE TITLE
A testing problem for validation set

### ISSUE THREAD

**Issue by PkuRainBow** [zhanghang1989/PyTorch-Encoding#83]
Hi, really sorry to bother again.

I am not very familiar with the testing settings for ADE20k, Pascal context and I find a small issue within your implementation,

```
    def _val_sync_transform(self, img, mask):
        outsize = self.crop_size
        short_size = outsize
        w, h = img.size
        if w > h:
            oh = short_size
            ow = int(1.0 * w * oh / h)
        else:
            ow = short_size
            oh = int(1.0 * h * ow / w)
        img = img.resize((ow, oh), Image.BILINEAR)
        mask = mask.resize((ow, oh), Image.NEAREST)
        # center crop
        w, h = img.size
        x1 = int(round((w - outsize) / 2.))
        y1 = int(round((h - outsize) / 2.))
        img = img.crop((x1, y1, x1+outsize, y1+outsize))
        mask = mask.crop((x1, y1, x1+outsize, y1+outsize))
        # final transform
        return img, self._mask_transform(mask)
```

According to the above implementation, you choose half center crop for evaluation on the validation set.
